phpunit.xml.dist

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<phpunit bootstrap="./tests/bootstrap.php" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" verbose="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" stderr="true" colors="true" verbose="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
Remove bootstrap file in favor of Pest.php (covered below) and ensure we're properly capturing stderr output.